psi4/src/psi4/libqt/qt.h

```diff
@@ -116,7 +116,7 @@ void C_DCOPY(size_t length, double *x, int inc_x,
              double *y, int inc_y);
 void C_DAXPY(size_t length, double a, double *x, int inc_x,
              double *y, int inc_y);
-double C_DDOT(size_t n, double *X, int inc_x, double *Y, int inc_y);
+PSI_API double C_DDOT(size_t n, double *X, int inc_x, double *Y, int inc_y);
```

Comment:
Try moving `PSI_API` to `blas_intfc.cc:230` where the function is implemented. It'll follow the pattern of the C-type functions you modified above.

Reply:
The pattern from the above is: `double PSI_API C_DDOT...`

Reply:
```
[ 15%] Building CXX object src/psi4/libqt/CMakeFiles/qt.dir/blas_intfc.cc.o
/home/psilocaluser/gits/hrw-patch/psi4/src/psi4/libqt/blas_intfc.cc(230): error: expected a ";"
  double PSI_API C_DDOT(size_t length, double *x, int inc_x, double *y, int inc_y)
                 ^

/home/psilocaluser/gits/hrw-patch/psi4/src/psi4/libqt/blas_intfc.cc(339): warning #12: parsing restarts here after previous syntax error
  }
  ^

compilation aborted for /home/psilocaluser/gits/hrw-patch/psi4/src/psi4/libqt/blas_intfc.cc (code 2)
make[5]: *** [src/psi4/libqt/CMakeFiles/qt.dir/blas_intfc.cc.o] Error 2
make[4]: *** [src/psi4/libqt/CMakeFiles/qt.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [psi4-core-prefix/src/psi4-core-stamp/psi4-core-build] Error 2
make[1]: *** [CMakeFiles/psi4-core.dir/all] Error 2
make: *** [all] Error 2
```

Reply:
Needs a `#include "psi4/pragma.h"` at the top of the cc file and then `double PSI_API C_DDOT` exports correctly.